app/templates/custom-elements/remote-screen.html

```diff
@@ -137,18 +137,20 @@
           if (!document.fullscreenElement) {
             this.fullscreen = false;
           } else {
-            const screenImg = this.shadowRoot.getElementById(
-              "remote-screen-img"
-            );
-            const windowRatio = window.innerWidth / window.innerHeight;
-            const screenRatio = screenImg.width / screenImg.height;
-            if (screenRatio > windowRatio) {
-              screenImg.classList.add("full-width");
-              screenImg.classList.remove("full-height");
-            } else {
-              screenImg.classList.add("full-height");
+            setTimeout(() => {
+              const screenImg = this.shadowRoot.getElementById(
+                "remote-screen-img"
+              );
               screenImg.classList.remove("full-width");
-            }
+              screenImg.classList.remove("full-height");
+              const windowRatio = window.innerWidth / window.innerHeight;
+              const screenRatio = screenImg.width / screenImg.height;
+              if (screenRatio > windowRatio) {
+                screenImg.classList.add("full-width");
+              } else {
+                screenImg.classList.add("full-height");
+              }
+            }, 250);
```

Comment:
Hmm, I'd like to avoid depending on an arbitrary timeout value. Could we detect this through listening for the `resize` event?

Reply:
Looking into that